Where2Play/Pages/Shared/_Layout.cshtml

```diff
@@ -13,7 +13,7 @@
     <header>
         <nav class="navbar navbar-expand-lg fixed-top">
             <div class="container-fluid">
-                <a class="navbar-brand me-auto" href="index">Where2Play</a>
+                <a class="navbar-brand me-auto" href="index"><img src="~/images/logo.jpg" width="100" /></a>
```

Comment:
img is missing alt text.   This is a point deduction if I see it in the final submission, as it is not accessibility friendly.
Describe the logo in concise terms.